apps/editor/src/wysiwyg/wwEditor.ts

```diff
@@ -171,13 +171,24 @@ export default class WysiwygEditor extends EditorBase {
             (ev as ClipboardEvent).clipboardData || (window as WindowWithClipboard).clipboardData;
           const items = clipboardData?.items;
 
-          if (items?.length === 1) {
-            const imageBlob = pasteImageOnly(items);
+          if (items) {
+            let containsRtf = false;
+            for (var i = 0; i < items.length; i++) {
+              const item = items[i];
+              if (item.kind === 'string' && item.type === 'text/rtf') {
+                containsRtf = true;
+                break;
+              }
+            }
```

Comment:
I think it would be simpler and readable to change it like below :)
```js
import { EditorView, NodeView } from 'prosemirror-view';
import { Node as ProsemirrorNode, Slice, Fragment, Mark, Schema } from 'prosemirror-model';
import isNumber from 'tui-code-snippet/type/isNumber';
// add below dependency
import toArray from 'tui-code-snippet/collection/toArray';

....
if (items) {
  const containRtfItem = toArray(items).some(item => item.kind === 'string' && item.type === 'text/rtf');
  ...
}
```